### PR TITLE
PR: Update `Phoenix` Version + Mention `fields` in `README.md` #36

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] add table of contents 📝 
+ [x] link to https://github.com/dwyl/fields #30 🔗 
+ [x] update **`Phoenix`** from `1.4` to `1.6` ⬆️ 
+ [x] update `:crypto` API #22 ₿
+ [x] add "Understanding Advanced Encryption Standard (AES)" section #24 + #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+[![docs](https://img.shields.io/badge/docs-maintained-brightgreen?style=flat-square)](https://hexdocs.pm/fields/api-reference.html)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/phoenix-ecto-encryption-example/issues)
 [![HitCount](https://hits.dwyl.com/dwyl/phoenix-ecto-encryption-example.svg)](https://hits.dwyl.com/dwyl/phoenix-ecto-encryption-example)
 
-## Why?
+</div>
+
+<br />
+
+💡 **Note**: we wrote this example/tutorial
```
Comment (nelsonic) [inline README.md:21]: this is the important bit #36 
please feel free to re-word it as you deem appropriate. 👍

**Comment by nelsonic** [comment]
cascade of failures because of newer `Erlang` removing `:crypto.block_encrypt/4` ... :facepalm:

**Comment by nelsonic** [comment]
Dang it. Updating this to repo from **`Phoenix`** `1.4` to `1.6` is turning into a bigger PR than I would have liked ... 🙄

**Comment by nelsonic** [comment]
@SimonLab  ref #36 + **`Phoenix`** update. Thanks! 🙏

**Review by SimonLab — APPROVED** [review]
LGTM, thanks for updating the Phoenix version

**Pull request merged**